build_projects/dotnet-host-build/PrepareTargets.cs

```diff
@@ -52,13 +52,13 @@ public static BuildTargetResult Init(BuildTargetContext c)
                     targetRID = $"win7-{RuntimeEnvironment.RuntimeArchitecture}";
                 }
             }
-            string targetFramework = Environment.GetEnvironmentVariable("TARGETFRAMEWORK") ?? "netcoreapp1.0";
+            string targetFramework = Environment.GetEnvironmentVariable("TARGETFRAMEWORK") ?? "netcoreapp1.1";
```

Comment:
I don't know what damian issue you are referring to so I cannot say if this will impact that or not.